### PR TITLE
Update modal styles

### PR DESCRIPTION
**Pull request by PavelLaptev**

### CODE REVIEW

File: packages/ui/src/lib/components/Modal.svelte [hunk 1]
```diff
@@ -13,59 +11,76 @@
...
 </script>
 
-<Overlay bind:this={modal} let:close on:close>
-	<div
-		class="flex cursor-default flex-col gap-y-4 py-4"
-		class:w-[680px]={width === 'default'}
-		class:w-[380px]={width === 'small'}
-		class:w-[980px]={width === 'large'}
-	>
-		{#if $$slots.title}
```
Comment (PavelLaptev) [inline packages/ui/src/lib/components/Modal.svelte:30]: title as a pop and icon as well

File: packages/ui/src/lib/components/Overlay.svelte [hunk 2]
```diff
@@ -21,24 +23,39 @@
 	}
 </script>
 
-<dialog class="dialog-overlay" bind:this={dialog} on:close={close} on:close>
+<dialog
+	class="dialog-overlay"
+	class:show-modal={open}
+	class:w-[680px]={width === 'default'}
+	class:w-[380px]={width === 'small'}
+	class:w-[860px]={width === 'large'}
```
Comment (mtsgrd) [inline packages/ui/src/lib/components/Overlay.svelte]: Can we make these rem units please?

File: packages/ui/src/lib/components/Overlay.svelte [hunk 3]
```diff
@@ -21,24 +23,39 @@
 	}
 </script>
 
-<dialog class="dialog-overlay" bind:this={dialog} on:close={close} on:close>
+<dialog
+	class="dialog-overlay"
```
Comment (mtsgrd) [inline packages/ui/src/lib/components/Overlay.svelte]: For a `<dialog>` element there is no explicit overlay. This element _is_ the dialog.

File: packages/ui/src/lib/components/Overlay.svelte [hunk 4]
```diff
@@ -21,24 +23,39 @@
...
 		background-color: var(--clr-theme-container-light);
+		border: 1px solid var(--clr-theme-container-outline-light);
+		box-shadow: var(--fx-shadow-l);
 
 		&::backdrop {
 			background-color: var(--clr-theme-overlay-bg);
 		}
 	}
+
+	.show-modal {
```
Comment (mtsgrd) [inline packages/ui/src/lib/components/Overlay.svelte]: What is the purpose of this class? The element is never visible unless it's open.

File: packages/ui/src/routes/[projectId]/board/+page.svelte [hunk 5]
```diff
@@ -70,7 +70,7 @@
 				/>
 			</div>
 		</div>
-		<Scrollbar {viewport} {contents} horz thickness="0.4rem" />
+		<Scrollbar {viewport} {contents} thickness="0.4rem" />
```
Comment (mtsgrd) [inline packages/ui/src/routes/[projectId]/board/+page.svelte]: This looks like a mistake?

File: packages/ui/src/styles/main.postcss [hunk 6]
```diff
@@ -41,84 +45,45 @@ button {
...
-	height: 8px;
-}
-
-.lane-scroll::-webkit-scrollbar-track {
-	@apply bg-light-200 dark:bg-dark-900;
-	-webkit-box-shadow: inset 0 0 0.0625rem rgba(0, 0, 0, 0.33);
-}
+/* We don't use REM here becasue we don't want
+the scrollbar to scale with the font size */
+.custom-scrollbar {
```
Comment (mtsgrd) [inline packages/ui/src/styles/main.postcss:51]: Should we maybe leave this out since it feels we do not intend to use it anywhere?

**Review by mtsgrd — APPROVED** [review]

File: packages/ui/src/styles/main.postcss (hunk 6 above)
Comment (PavelLaptev) [reply packages/ui/src/styles/main.postcss:51]: I'll remove this later. In another PR

**Pull request merged**